src/theme/editor.ts

```diff
@@ -0,0 +1,357 @@
+      token: 'support.constant.media-type',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'support.constant.media',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'constant.other.color.rgb-value',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'constant.other.rgb-value',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'support.constant.color',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'punctuation.definition.group.regexp',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'punctuation.definition.group.assertion.regexp',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'punctuation.definition.character-class.regexp',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'punctuation.character.set.begin.regexp',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'punctuation.character.set.end.regexp',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'keyword.operator.negation.regexp',
+    },
+    {
+      foreground: '#CE9178',
+      token: 'support.other.parenthesis.regexp',
+    },
+    {
+      foreground: '#d16969',
+      token: 'constant.character.character-class.regexp',
+    },
+    {
+      foreground: '#d16969',
+      token: 'constant.other.character-class.set.regexp',
+    },
+    {
+      foreground: '#d16969',
+      token: 'constant.other.character-class.regexp',
+    },
+    {
+      foreground: '#d16969',
+      token: 'constant.character.set.regexp',
+    },
+    {
+      foreground: '#DCDCAA',
+      token: 'keyword.operator.or.regexp',
+    },
+    {
+      foreground: '#DCDCAA',
+      token: 'keyword.control.anchor.regexp',
+    },
+    {
+      foreground: '#d7ba7d',
+      token: 'keyword.operator.quantifier.regexp',
+    },
+    {
+      foreground: '#569cd6',
+      token: 'constant.character',
+    },
+    {
+      foreground: '#d7ba7d',
+      token: 'constant.character.escape',
+    },
+    {
+      foreground: '#C8C8C8',
+      token: 'entity.name.label',
+    },
+    {
+      foreground: '#569CD6',
+      token: 'constant.language',
+    },
+    {
+      foreground: '#569CD6',
+      token: 'entity.name.tag',
+    },
+    {
+      foreground: '#569cd6',
+      token: 'storage',
+    },
+  ],
+  colors: {
+    'editor.foreground': semanticColors.text,
+    'editor.errorForeground': semanticColors['text-error'],
+    'editor.descriptionForeground': semanticColors['text-xlight'],
+    'editor.background': semanticColors['fill-one'],
+    'editor.lineHighlightBackground': semanticColors['fill-one-hover'],
+    'editor.selectionBackground': semanticColors['fill-one-selected'],
+    'editor.inactiveSelectionBackground': semanticColors['fill-one-hover'],
+    'editorCursor.foreground': semanticColors.text,
+    'editorLineNumber.foreground': semanticColors.text,
+    'scrollbarSlider.background': semanticColors['fill-three'],
+    'scrollbarSlider.hoverBackground': semanticColors['fill-three'],
+  },
+}
```

Comment:
@nickcernera @cris-dobbins this file contains definition of theme that I've used. You can check how it looks in preview. 

Tokens are responsible for code highlighting, colors are related to general editor styling. You can find more available colors here:

- https://github.com/microsoft/monaco-editor/issues/1631
- https://microsoft.github.io/monaco-editor/playground.html#customizing-the-appearence-exposed-colors
- https://code.visualstudio.com/api/references/theme-color#editor-colors

Do you have any ideas for improvements that we should apply before adding it to console?

Reply:
List of other options like minimap etc.: https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IEditorConstructionOptions.html#lineDecorationsWidth